tests/utils.go

```diff
@@ -807,3 +809,22 @@ func NewConsoleExpecter(virtCli kubecli.KubevirtClient, vm *v1.VirtualMachine, c
 		Check: func() bool { return true },
 	}, timeout, opts...)
 }
+
+type RegistryDisk string
+
+const (
+	CirrosRegistryDisk RegistryDisk = "cirros"
```

Comment:
Can you change all constants to form
RegistryDiskCirros
RegistryDiskAlpine
it just more easy to view all registry disk with auto-completion

Reply:
Done. :+1: